src/components/charts/LineChart.tsx

```diff
@@ -0,0 +1,185 @@
+import React from "react"
+import merge from "deepmerge"
+
+import {
+  ChartWrapper,
+  RecordSet,
+  RecordEntry,
+  ChartProps,
+} from "./ChartWrapper"
+
+export interface LineChartProps extends ChartProps {
+  strokeWidths?: number | number[]
+  /**
+   * The type of dashed lines for each line. Higher numnbers means more space between dashes.
+   */
+  dashTypes?: number | number[]
+  /**
+   * Whether each line is curved.
+   */
+  curved?: boolean
+  /**
+   * Specify the size of markers for each line. 0 means no markers.
+   */
+  markerSizes?: number | number[]
+  /**
+   * The title of the chart
+   */
+  xMin?: number
+  /**
+   * The maximum value of the x (horizontal) or y axis. Useful when data labels overflow.
+   */
+  xMax?: number
+  /**
+   * The minimum value of the x (horizontal) or y axis.
+   */
+  yMin?: number
+  /**
+   * The maximum value of the x (horizontal) or y axis. Useful when data labels overflow.
```

Comment:
You are referencing a property (`horizontal`) that is neither contained in this interface nor in the parent interface. I'd suggest dropping this for now and instead adding a note to the interface that introduces the `horizontal` property as to what it does.

Reply:
Fixed it. This is the result of copying and pasting without changing the content lol